content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If co-workers and customers depend on Kubernetes being available to run critical
+workloads, it is your job to make sure that the platform is set up and
+maintained to meet their demands.
+
+<!-- body -->
+
+## Production issues
+
+A personal Kubernetes learning or development environment is not meant to be
+used in production. A production environment may require secure access by many
+users, consistent availability, and the resources to adapt to changing demands.
```

Comment:
```suggestion
Typically, a production environment Kubernetes has more requirements than a development and test environment Kubernetes. A production environment may require secure access by many users, consistent availability, and the resources to adapt to changing demands.
```
I think it's obvious that a personal or learning Kubernetes is not meant for production. Typically I find end-users have a development and/or test environment Kubernetes in addition to their production environment.

Reply:
Okay. I used your wording.